src/components/Form.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Button from './Button';
+import FormInput from './FormInput';
+
+import './Form.css';
+
+function Form({ form, setForm, onSubmit }) {
+  function setTerms(e) {
+    setForm({ ...form, terms: !e.target.defaultChecked });
```

Comment:
`defaultChecked` veranderd nooit. Als je dus de state van de checkbox wilt overnemen in je form object dan moet je het zo doen: `setForm({ ...form, terms: e.target.checked });`